x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/planner/QueryFolderOkTests.java

```diff
@@ -45,18 +45,13 @@
             },
             {"inFilter", "process where process_name in (\"python.exe\", \"SMSS.exe\", \"explorer.exe\")",
                     new Object[]{
-                            "\"term\":{\"process_name\":{\"value\":\"python.exe\"",
-                            "\"term\":{\"process_name\":{\"value\":\"SMSS.exe\"",
-                            "\"term\":{\"process_name\":{\"value\":\"explorer.exe\"",
+                            "{\"terms\":{\"process_name\":[\"python.exe\",\"SMSS.exe\",\"explorer.exe\"],\"boost\":1.0}}],\"boost\":1.0}}",
                     }
             },
             {"equalsAndInFilter", "process where process_path == \"*\\\\red_ttp\\\\wininit.*\" and opcode in (0,1,2,3)",
                     new Object[]{
                             "\"wildcard\":{\"process_path\":{\"wildcard\":\"*\\\\\\\\red_ttp\\\\\\\\wininit.*\"",
-                            "\"term\":{\"opcode\":{\"value\":0",
-                            "\"term\":{\"opcode\":{\"value\":1",
-                            "\"term\":{\"opcode\":{\"value\":2",
-                            "\"term\":{\"opcode\":{\"value\":3",
+                            "{\"terms\":{\"opcode\":[0,1,2,3],\"boost\":1.0}}],\"boost\":1.0}}",
```

Comment:
I would just check the terms and opcode not the boost - that's a default we don't influence.